editor/lib/display/header_view.dart

```diff
@@ -111,7 +114,26 @@ class _HeaderViewState extends State<HeaderView> {
   Future<void> _importConfigs() async {
     final configs = await browseConfigs(["json"]);
     if (configs != null) {
-      widget.controller.addLayer(configsData: configs);
+      widget.controller.addConfigLayer(configsData: configs);
+    }
+  }
+
+  /// Browse and import configs
+  Future<void> _importConfigsWithTextures() async {
+    final files = await browseConfigsWithTexture();
+    for (var entry in files.entries) {
+      if (!entry.key.endsWith(".json")) {
+        continue;
+      }
+      List configLayers = entry.value;
+      for (var configLayer in configLayers) {
+        final layer = ParticularEditorLayer(
+          texture: files[configLayer["textureFileName"]].$1,
+          textureBytes: files[configLayer["textureFileName"]].$2,
+          configs: ParticularConfigs()..initialize(configs: configLayer),
```

Comment:
ParticularConfigs()..initialize(configs: configLayer) change to factory method for directly use ParticularConfigs.initialize

Reply:
Done